edge/pkg/metamanager/apiserver-lite/kubernetes/storage/cacher/cacher.go

```diff
@@ -0,0 +1,52 @@
+package cacher
+
+import (
+	"fmt"
+	"github.com/kubeedge/kubeedge/edge/pkg/metamanager/apiserver-lite/kubernetes/storage/sqlite"
+	"github.com/kubeedge/kubeedge/pkg/apiserverlite"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apiserver/pkg/storage"
+	cacherstorage "k8s.io/apiserver/pkg/storage/cacher"
+	"k8s.io/apiserver/pkg/storage/etcd3"
+)
+func UnstrIndexFunc(obj interface{}) ([]string, error) {
+	unstrObj, ok := obj.(*unstructured.Unstructured)
+	if !ok {
+		return nil, fmt.Errorf("not a unstructured obj")
+	}
+	key,err := apiserverlite.KeyFuncObj(unstrObj)
+	if err !=nil{
+		return []string{},err
+	}
+	return []string{key},nil
+}
+func NewCacher() (storage.Interface,error){
+	cacherConfig :=cacherstorage.Config{
+		Storage:        sqlite.New(),
+		Versioner:      etcd3.Versioner,
+		ResourcePrefix: "",
+		KeyFunc:        apiserverlite.KeyFuncObj,
+		NewFunc: func()runtime.Object{
+			unstr := unstructured.Unstructured{}
+			return &unstr
+		},
+		NewListFunc: func()runtime.Object{
+			unstrList := unstructured.UnstructuredList{}
+			return &unstrList
+		},
+		GetAttrsFunc: storage.DefaultNamespaceScopedAttr,
+		//IndexerFuncs: map[string]storage.IndexerFunc{"metadata.name": configmap.NameTriggerFunc},
```

Comment:
?

Reply:
temporarily set to nil, need to figure out a appropriate indexerfunc